spring-cloud-kubernetes-fabric8-autoconfig/src/main/java/org/springframework/cloud/kubernetes/fabric8/Fabric8AutoConfiguration.java

```diff
@@ -117,4 +119,15 @@ public Fabric8PodUtils kubernetesPodUtils(KubernetesClient client) {
 		return new Fabric8PodUtils(client);
 	}
 
+	@PreDestroy
```

Comment:
Can we just add `@Bean(destroyMethod = "close")` to the `KubernetesClient` bean above?

Reply:
I'm guessing that didn't work?